src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --header-secondary-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```

Comment:
it is going by default, no needs to pass it directly
```suggestion
```

Reply:
Oh, got it... Thanks!